### PR TITLE
This branch is for creating contact form in the mobile version of My-portfolio

### PR DESCRIPTION
**Pull request by thenarain**
Add files: 

- Update Markup
- Update styles
- Add Images

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -234,6 +232,39 @@ <h3>Skills</h3>
...
+          </li>
+          <li>
+            <label for="msg"></label>
+            <textarea class="form-list-textarea" id="msg" name="user_message" placeholder="Write your message here"
+              maxlength="500" required></textarea>
+          </li>
+          <li class="button-list">
+            <button class="button1" type="submit">Get It Touch</button>
+          </li>
+        </ul>
```
Comment (Cathella) [inline index.html:264]: Hello @thenarain, You have done a tremendous job aligning the form elements as required 💪🏻! However, I have a few comments to make that may lead to changes 😄.

- [ ] I have noticed that the inputs and the textarea need some padding on the left to improve on how they appear like in the figma design. Please checkout the comparison below 👇🏻 

| Figma design | Your design |
| --- | --- |
| image | image |

- [ ] In the form body, I have also noticed that the button's background differs from that in the figma design. In reference to the project requirements, you are encouraged to design this portfolio form to almost look exactly like the one designed in figma. Please see the comparison below 👇🏻 

| Figma design | Your design |
| --- | --- |
| image | image |

**Review by Cathella — CHANGES_REQUESTED** [review]
## 🟡 Status: Changes Required 🛑 

### Hi @thenarain 👋🏻, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## To Highlight 👏🏻 

✅ No linter errors.
✅  You used correct Github Flow.
✅  Form uses the action and method attribute to integrate with formspree service.
✅ Form works as expected/intended 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (thenarain) [reply index.html:264]: Fixed both the issues.

**Review by elmar8287 — APPROVED** [review]
Hi, 

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 

**OPTIONAL suggestions:**
- Please, consider the button `Get in touch` active, hover, ... styling.
> Here is your version


> The Figma design
Screen Shot 2022-01-23 at 00 43 23


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**